website/common/locales/en@pirate/messages.json

```diff
@@ -30,6 +30,7 @@
     "messageAlreadyPurchasedGear": "Ye purchased this gear in th' past, but do not currently own it. Ye can buy it again in th' rewards column on th' tasks page.",
     "messageAlreadyOwnGear": "Ye already own this item. Equip it by going to th' equipment page.",
     "messageHealthAlreadyMax": "You already have maximum health.",
+    "messageHealthAlreadyMin": "You already have the minimum health.",
```

Comment:
Sure, removed.
Will double check the README to see if I can contribute with some en/es translations.